android/console.c

```diff
@@ -3523,12 +3523,44 @@ do_modem_tech( ControlClient client, char* args )
     return 0;
 }
 
+static int
+do_modem_radio( ControlClient client, char *args )
+{
+    if (!client->modem) {
+        control_write(client, "KO: modem emulation not running\r\n");
+        return -1;
+    }
+
+    if (!args) {
+        ARadioState state = amodem_get_radio_state(client->modem);
```

Comment:
This is for querying current state, you have to write current radio state to console via ```control_write```.